pwa-devdocs/src/technologies/tools-libraries/index.md

```diff
@@ -2,15 +2,41 @@
 title: Tools and libraries
 ---
 
+![tools and technology overview][]
+
 To work with the utilities provided by the Magento PWA Studio project, you should be familiar with the tools and libraries listed in the following table:
 
-|                     |                  Contributor                  |              Frontend developer               |       Sysadmin/Devops/Backend developer       |
-| ------------------- | :-------------------------------------------: | :-------------------------------------------: | :-------------------------------------------: |
-| [webpack](#webpack) | <i class="material-icons green">check_box</i> |                                               |                                               |
-| [React](#react)     | <i class="material-icons green">check_box</i> | <i class="material-icons green">check_box</i> |                                               |
-| [Redux](#redux)     | <i class="material-icons green">check_box</i> | <i class="material-icons green">check_box</i> |                                               |
-| [GraphQL](#graphql) | <i class="material-icons green">check_box</i> | <i class="material-icons green">check_box</i> | <i class="material-icons green">check_box</i> |
-{:style="table-layout:auto"}
+|                           |                  Contributor                  |              Frontend developer               |         Sysadmin/Devops/Backend developer          |
```

Comment:
What's the difference between a contributor and a front end developer?

Reply:
I see a contributor as also having backend scripting skills to work on packages such buildpack or upward-js